guides/Framework/CakePHP

```diff
@@ -0,0 +1,9 @@
+# CakePhp
+
+## What is it ? 
+
+This is an open source php Framework based on MVC pattern. Until MIT licence.  
```

Comment:
'Cake' is an open source, sounds better than this is an open source

Reply:
@penguinpet , quoting from the `CONTRIBUTING.md`:
>The maintainer will review your PR in an span of 2 days. Do not review other contributor's PR until said so by the maintainer himself.
Please don't review PRs until said so by the maintainer(me)